src/layouts/Header/components/Navigator/MyContextSwitcher.tsx

```diff
@@ -21,6 +22,9 @@ function MyContextSwitcher({ pageName }: MyContextSwitcherProps) {
     isLoading,
   } = useMyContexts({ provider })
   const { data: activeContext } = useOwner({ username: owner })
+  const memoizedOnLoadMore = useCallback(() => {
+    hasNextPage && fetchNextPage()
+  }, [hasNextPage, fetchNextPage])
```

Comment:
Was this causing re-render issues?

Reply:
I didn't test how many times it was rerendering but made this change to avoid the anonymous function trigger rerenders. Did you have thoughts about not making this change?